ui/src/app/machines/components/ActionFormWrapper/ActionFormWrapper.tsx

```diff
@@ -45,13 +45,13 @@ const getErrorSentence = (action: SelectedAction, count: number) => {
     case NodeActions.UNLOCK:
       return `${machineString} cannot be unlocked`;
     default:
-      return `${machineString} cannot be ${action.sentence}`;
+      return `${machineString} cannot be ${action.extras?.sentence}`;
```

Comment:
I wonder if we should actually just have a case for each action instead of relying on the api for this, since they're inconsistent with tenses. Followup maybe?

Reply:
Thanks, I've added an issue: https://github.com/canonical-web-and-design/maas-ui/issues/2730.